templates/admin/applications/edit.tmpl

```diff
@@ -0,0 +1,55 @@
+{{template "base/head" .}}
+<div class="page-content admin config">
+	{{template "admin/navbar" .}}
+	<div class="ui container">
```

Comment:
Can you reuse templates https://github.com/go-gitea/gitea/blob/main/templates/user/settings/applications_oauth2_edit_form.tmpl  and https://github.com/go-gitea/gitea/blob/main/templates/user/settings/applications_oauth2_list.tmpl ?